### DISCUSSION TITLE
Auto loading batch of image textures to Bonsai

### DISCUSSION THREAD

**Discussion by ThetaS** [bonsai-rx/bonsai#1123] [Q&A]
Hi all,

I'm finding a way to load a set of image stimuli to Bonsai automatically so that I don't need to manually do it from the managing window of Texture Resources. In the ideal scenario it can pull and load from the list of directories I put in a CSV file. May I have any suggestions on how this could be done? Thank you!

**Comment by bruno-f-cruz** [comment]
Heya,
This might work:
https://groups.google.com/g/bonsai-users/c/HRYyQqg5fhw/m/UgVDk4ohCwAJ
cheers,
B

**Reply by ThetaS** [reply]
Hi Bruno,

Thanks for the response.  In the tutorial you gave, I understood the part you load the image and bind them to texture. But how to use the texture collection？ Can draw image or draw mesh use this collection? Thanks.

PS: Our image is stored in numpy (npy) format. So I wonder if there's a method to directly load from that as well.

Best,
